qe_get_tendrl_ansible.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+#
+# Get content of tendrl-ansible package
+#
+
+- hosts: usm_server
+  remote_user: root
+  vars:
+    tendrl_ansible_dir: "./tendrl-ansible-tmp/"
+  roles:
+    - qe-tendrl-repo
+  tasks:
+    # This task is not necessary, but we will at least check that
+    # installation works as expected
+    - name: Install tendrl-ansible package
+      yum: name=tendrl-ansible state=latest
```

Comment:
It would be better to have a separate test case for this. One part of the installation is already covered in yum install rpm test cases, but I agree that there definitely is a room for more testing. I would suggest to draft a test case with some *actual checks* instead of this.

Reply:
I created an issue for that here: https://github.com/usmqe/usmqe-testdoc/issues/13

Reply:
My intention with installing the tendrl-ansible package on the Tendrl server is mainly to *have it installed* somewhere - for any kind of manual tests/checks/work... (It shouldn't break anything and as a side effect it check, if the installation works.)
I totally agree, that we should test it separately and more deeply.